qemu/tests/rng_maxbytes_period.py

```diff
@@ -0,0 +1,85 @@
+import re
+import logging
+
+from virttest import error_context
+from virttest import env_process
+from virttest import virt_vm
+from virttest import utils_misc
+
+from aexpect.exceptions import ShellTimeoutError
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu virtio-rng device test:
+    1) boot guest with virtio-rng device
+    2) read random data in guest
+    3) check the read data rate
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    def _is_rngd_running():
+        """
+        Check whether rngd is running
+        """
+        output = session.cmd_output(check_rngd_service)
+        if 'running' not in output:
```

Comment:
return 'running' in output

is the keyword 'running' enough for the checking? Do we need to check more , e.g. Active: active (running)

Reply:
Yes, it's enough. 'running' is the core keyword in output. Thanks!

Reply:
what about 
return 'running' in output

Reply:
Updated, thanks!